src/dispatching/dialogue/mod.rs

```diff
@@ -190,11 +190,11 @@ where
 /// (This next handler can be an endpoint or a more complex one.) The payload
 /// format depend on the form of `MyVariant`:
 ///
-///  - For `State::MyVariant(param)`, the payload is `param`.
-///  - For `State::MyVariant(param1, ..., paramN)`, the payload is `(param1,
-///    ..., paramN)` (where `N` > 1).
-///  - For `State::MyVariant { param1, ..., paramN }`, the payload is `(param1,
-///    ..., paramN)`.
+///  - For `State::MyVariant(param)` and `State::MyVariant { param }`, the
+///    payload is `param`.
+///  - For `State::MyVariant(param1, ..., paramN)` and `State::MyVariant {
+///    param1, ..., paramN }`, the payload is `(param1, ..., paramN)` (where `N`
+///    > 1).
```

Comment:
`N` can be `1`, documentation is misleading.

Reply:
Fixed.